plugins/hotkeys.cpp

```diff
@@ -90,7 +90,7 @@ static void find_active_keybindings(df::viewscreen *screen, bool filtermenu) {
 
     valid_keys.push_back("`");
 
-    current_focus = Gui::getFocusString(screen);
+    current_focus = Gui::getFocusStrings(screen)[0];
```

Comment:
this is the same algorithm as the keybinding match, essentially. it will need all the focus strings, and will need to match any of them down on line 115